Examples/hello-world-hummingbird-server-otlp-http-json/docker/docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: "3.5"
+services:
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:latest
+    command: ["--config=/etc/otel-collector-config.yaml"]
+    volumes:
+      - ./otel-collector-config.yaml:/etc/otel-collector-config.yaml
+      - ./logs:/logs:rw
+    ports:
+      - "4317:4317"  # OTLP/gRPC receiver
```

Comment:
Do we want to stop exposing the gRPC port here to make it a bit clearer that we only need the HTTP port for this example?